components/abtesting/optimizelyXExperiment/src/optimizely-x.js

```diff
@@ -0,0 +1,65 @@
+import ExperimentUseCase from './use-case'
+
+class OptimizelyXExperimentsService {
+  /**
+   * Get OptimizelyX SDK instance
+   * @return Promise<Object>
+   */
+  static async getSDK () {
+    return window && window.optimizely
+  }
+
+  /**
+   * Get info of an experiment
+   * @param {String} experimentId
+   * @return Promise<Object>
+   * @example
+   * {
+   *  id: '100',
+   *  isActive: true,
+   *  isInExperimentHoldback: true,
+   *  variation: {
+   *      id: '101',
+   *      name: 'Variation #1'
+   *  }
+   * }
+   */
+  static async getInfo (experimentId) {
+    let sdk = await this.getSDK()
```

Comment:
When you say it's a mutable object, it seems that you're changing the value of the variable and I would say that you're not doing it. Making it a `const` would give you an error in the linter if you try to reassign the `sdk` when you already know is something you want to avoid. It's a effortless change that could avoid problems. :P